docs/creating-and-editing-specialist-document-types.md

```diff
@@ -70,18 +70,18 @@ The email sign up page is rendered by [Finder Frontend](https://github.com/alpha
 
 If your email sign up page should have checkboxes (e.g. [cma-cases](https://www.gov.uk/cma-cases/email-signup)), you will need to edit email-alert-api by adding the new tags to [valid_tags.rb](https://github.com/alphagov/email-alert-api/blob/3e0018510ea85f5d561e2865ad149832b94688a1/lib/valid_tags.rb#L2).
 
-## 5. Deploy and publish
+## 5. Deploy and publish <a id="deploy-and-publish"></a>
```

Comment:
@baisa Apparently markdown does not support referencing an anchor which is a heading and a list item. This works fine now ;)